docs/accessibility.md

```diff
@@ -0,0 +1,58 @@
+# Accessibility
+
+## Colors
+
+Treeherder uses [Bootstrap color](https://getbootstrap.com/docs/4.0/utilities/colors/) utility classes. While most of them work, there are a couple that needed a little tweaking in order to increase **color contrast**, according to [WCAG standards](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast?utm_source=devtools&utm_medium=a11y-panel-checks-color-contrast).
+
+So, we ask that you substitute the Bootstrap color to the Treeherder one, according to the following table:
+
+| Bootstrap Color | Treeherder Color       |
+| --------------: | :--------------------- |
+|      `btn-info` | `btn-darker-info`      |
+|     `text-info` | `text-darker-info`     |
+| `btn-secondary` | `btn-darker-secondary` |
+
+For example, if inserting a text color into an element, use `className="text-darker-secondary"`.
+
+### Colors in buttons
+
+In [`reactstrap`'s `<Button>` component](https://reactstrap.github.io/components/buttons/), you can insert a `prop` specifying its color. In that case, you use only the color name, without its prefix.
+
+| Treeherder Color | Button prop color  |
+| ---------------: | :----------------- |
+|       `btn-info` | `darker-info`      |
+|  `btn-secondary` | `darker-secondary` |
+
+For example:
+
+```jsx
+<Button color="darker-info">
+  Info colored button with contrast passing score
+</Button>
+```
+
+### Inserting new colors
+
+If you are inserting new colors into the Treeherder UI, please make sure the color + background-color combination passes the minimum constrast ratio defined by WCAG.
```

Comment:
"If you add new colors to the style sheets, please make sure the text and background color combination passes the minimum contrast ratio defined by WCAG."

(also you have a typo with contrast).